### PR TITLE
maintainers: Modify format of file

### PR DESCRIPTION
**Pull request by antonio-nino-diaz-arm**
The previous format was not very practical and hard to parse by scripts. The current format is easier as it uses more tokens that can be identified by scripts, while maintaining a reasonably good result when it is rendered.

Some maintainers maintain more than one subsystem but they were all part of the same entry. In cases like this the entry has been split into two to clarify what file belongs to which subsystem.

The list of maintainers of the Trusted Firmware has been updated.

### CODE REVIEW

**Comment by antonio-nino-diaz-arm** [comment]
Note that GitHub doesn't render rst files in the same way as other viewers such as http://rst.ninjs.org/

**Comment by antonio-nino-diaz-arm** [comment]
@hzhuang1, @vwadekar, @jenswi-linaro, you maintain more than one subsystem so I've split your entries into two. I can put them back into one if you want, but I think it's cleaner this way.

**Comment by jenswi-linaro** [comment]
Looks good to me.
